hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -252,8 +253,9 @@ private void enablePartition(final MetadataPartitionType partitionType, final Ho
    * @param writeConfig                {@code HoodieWriteConfig} of the main dataset writer
    * @param failedWritesCleaningPolicy Cleaning policy on failed writes
    */
-  private HoodieWriteConfig createMetadataWriteConfig(
+  public static HoodieWriteConfig createMetadataWriteConfig(
```

Comment:
can we move static methods to HoodieTableMetadataUtil ?

Reply:
It cannot be moved to `HoodieTableMetadataUtil` in `hudi-common` module since some classes in client modules are referenced.  I created a new `HoodieMetadataWriteUtils` to which the method is moved.